src/components/ebay-video/component.ts

```diff
@@ -79,13 +89,15 @@ interface State {
     isLoaded: boolean;
     volumeSlider: boolean;
     action: "play" | "pause" | "";
-    showLoading: boolean;
 }
 
 class Video extends Marko.Component<Input, State> {
     declare video: HTMLVideoElement;
     declare root: HTMLElement;
     declare containerEl: HTMLElement;
+    declare spacer: HTMLElement;
+    declare rangeContainer: HTMLElement;
+    declare buttonPanel: HTMLElement;
```

Comment:
These are all only used in one function, why keep them as class members? Will they ever be shared between methods?